CodeEdit/WorkspaceView.swift

```diff
@@ -23,15 +23,20 @@ struct WorkspaceView: View {
     @State private var showingAlert = false
     @State private var alertTitle = ""
     @State private var alertMsg = ""
+    @State var showInspector = true
 
     var body: some View {
         NavigationView {
             if workspace.workspaceClient != nil {
-                SideBar(workspace: workspace, windowController: windowController)
+                NavigatorSidebar(workspace: workspace, windowController: windowController)
                     .frame(minWidth: 250)
-
-                WorkspaceCodeFileView(windowController: windowController,
-                                      workspace: workspace)
+                HSplitView {
+                    WorkspaceCodeFileView(windowController: windowController,
+                                    workspace: workspace)
+              .frame(maxWidth: .infinity, maxHeight: .infinity)
+                  InspectorSidebar(workspace: workspace, windowController: windowController)
+                  .frame(minWidth: 250, maxWidth: .infinity, maxHeight: .infinity)
```

Comment:
using fixed values worries me a little bit... maybe 250 it's low and not that much but we should definitely put this value inside a constant that maybe is injected upon initalization?